### PR TITLE
fix: [#9666] Onclick Insert property reference in a memory drop down not appearing in Bot Framework Composer 2.1.3.

### LINKED ISSUE #9666: Onclick Insert property reference in a memory drop down not appearing in Bot Framework Composer 2.1.3.
## Describe the bug

Onclick Insert property reference in a memory drop down not appearing in Bot Framework Composer 2.1.3. Works in 2.1.2. Not sure about previous nightly versions.

## Version

Version:  2.1.3-nightly.364675.1a93f21
Electron: 14.2.6
Chrome: 93.0.4577.82
NodeJS: 14.17.0
V8: 9.3.345.20-electron.0

## Browser

- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## To Reproduce

1. Set a property or Set Properties in a dialog or trigger
2. set property variable/scope
3. Use an expression for the value
4. See that the drop down for Insert a property reference in memory (as well as Insert an adaptive expression...) is greyed out and unusable.

## Expected behavior

Should be able to click and select property references

## Screenshots

image

### PR DESCRIPTION
**Pull request by JhontSouth**
## Description
This PR adds the CSRF token in the headers of the _fetch_ request of the project memory variables.

## Task Item

Fixes #9666

## Screenshots
The following image shows the references dropdown working after the fix.
image

### CODE REVIEW

**Comment by tracyboehrer** [comment]
image

**Comment by tracyboehrer** [comment]
Build error "The lockfile would have been modified by this install, which is explicitly forbidden."

**Comment by JhontSouth** [comment]
> Build error "The lockfile would have been modified by this install, which is explicitly forbidden."

Thank you Tracy, we are reviewing it.

File: Composer/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "serialize-javascript": "^3.1.0",
     "set-value": "^3.0.2",
     "terser-webpack-plugin": "^2.3.7",
-    "axios": "^0.21.1",
+    "axios": "^0.22.0",
```
Comment (OEvgeny) [inline Composer/package.json:18]: This needs either to include updated lock files or exluding the change from the PR

**Comment by tracyboehrer** [comment]
Summary of all failing tests
FAIL packages/client/__tests__/pages/language-generation/LGPage.test.tsx
  ● LG page all up view › should render lg page code editor

    TypeError: Cannot read property 'cancel' of undefined

      102 |     // clean up pending async request
      103 |     return () => {
    > 104 |       source.cancel();
          |              ^
      105 |     };
      106 |   }, [projectId]);
      107 |

      at src/pages/language-generation/code-editor.tsx:104:14
      at HTMLUnknownElement.callCallback (../../node_modules/react-dom/cjs/react-dom.development.js:188:14)
      at HTMLUnknownElement.callTheUserObjectsOperation (../../node_modules/jsdom/lib/jsdom/living/generated/EventListener.js:26:30)
      at innerInvokeEventListeners (../../node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:338:25)
      at invokeEventListeners (../../node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
      at HTMLUnknownElementImpl._dispatch (../../node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
      at HTMLUnknownElementImpl.dispatchEvent (../../node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
      at HTMLUnknownElement.dispatchEvent (../../node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:231:34)
      at Object.invokeGuardedCallbackDev (../../node_modules/react-dom/cjs/react-dom.development.js:237:16)
      at invokeGuardedCallback (../../node_modules/react-dom/cjs/react-dom.development.js:292:31)
      at safelyCallDestroy (../../node_modules/react-dom/cjs/react-dom.development.js:19617:5)
      at ../../node_modules/react-dom/cjs/react-dom.development.js:20090:21
      at unstable_runWithPriority (../../node_modules/scheduler/cjs/scheduler.development.js:653:12)
      at runWithPriority$1 (../../node_modules/react-dom/cjs/react-dom.development.js:11039:10)
      at commitUnmount (../../node_modules/react-dom/cjs/react-dom.development.js:20083:15)
      at unmountHostComponents (../../node_modules/react-dom/cjs/react-dom.development.js:20464:7)
      at commitDeletion (../../node_modules/react-dom/cjs/react-dom.development.js:20500:5)
      at commitMutationEffects (../../node_modules/react-dom/cjs/react-dom.development.js:22782:11)
      at HTMLUnknownElement.callCallback (../../node_modules/react-dom/cjs/react-dom.development.js:188:14)
      at HTMLUnknownElement.callTheUserObjectsOperation (../../node_modules/jsdom/lib/jsdom/living/generated/EventListener.js:26:30)
      at innerInvokeEventListeners (../../node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:338:25)
      at invokeEventListeners (../../node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
      at HTMLUnknownElementImpl._dispatch (../../node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
      at HTMLUnknownElementImpl.dispatchEvent (../../node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
      at HTMLUnknownElement.dispatchEvent (../../node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:231:34)
      at Object.invokeGuardedCallbackDev (../../node_modules/react-dom/cjs/react-dom.development.js:237:16)
      at invokeGuardedCallback (../../node_modules/react-dom/cjs/react-dom.development.js:292:31)
      at commitRootImpl (../../node_modules/react-dom/cjs/react-dom.development.js:22509:9)
      at unstable_runWithPriority (../../node_modules/scheduler/cjs/scheduler.development.js:653:12)
      at runWithPriority$1 (../../node_modules/react-dom/cjs/react-dom.development.js:11039:10)
      at commitRoot (../../node_modules/react-dom/cjs/react-dom.development.js:22381:3)
      at finishSyncRender (../../node_modules/react-dom/cjs/react-dom.development.js:21807:3)
      at performSyncWorkOnRoot (../../node_modules/react-dom/cjs/react-dom.development.js:21793:7)
      at scheduleUpdateOnFiber (../../node_modules/react-dom/cjs/react-dom.development.js:21188:7)
      at updateContainer (../../node_modules/react-dom/cjs/react-dom.development.js:24373:3)
      at legacyRenderSubtreeIntoContainer (../../node_modules/react-dom/cjs/react-dom.development.js:2[477](https://github.com/microsoft/BotFramework-Composer/actions/runs/7264840533/job/19795205920?pr=9678#step:9:478)4:5)
      at ../../node_modules/react-dom/cjs/react-dom.development.js:2[488](https://github.com/microsoft/BotFramework-Composer/actions/runs/7264840533/job/19795205920?pr=9678#step:9:489)4:7
      at unbatchedUpdates (../../node_modules/react-dom/cjs/react-dom.development.js:21903:12)
      at Object.unmountComponentAtNode (../../node_modules/react-dom/cjs/react-dom.development.js:24883:5)
      at ../../node_modules/@testing-library/react/dist/pure.js:144:23
      at batchedUpdates$1 (../../node_modules/react-dom/cjs/react-dom.development.js:21856:12)
      at act (../../node_modules/react-dom/cjs/react-dom-test-utils.development.js:929:14)
      at cleanupAtContainer (../../node_modules/@testing-library/react/dist/pure.js:143:26)
          at Set.forEach (<anonymous>)
      at cleanup (../../node_modules/@testing-library/react/dist/pure.js:137:21)
      at Object.<anonymous> (../../node_modules/@testing-library/react/dist/index.js:33:25)

**Comment by JhontSouth** [comment]
@tracyboehrer , @OEvgeny 
We reverted the _axios_ updates and kept the _fetch_ request, but now we're using the CSRF token in the request headers.
The PR is ready for review now.

**Review by OEvgeny — APPROVED** [review]

**Pull request merged**